tests/components/airos/test_config_flow.py

```diff
@@ -0,0 +1,239 @@
+"""Test the Ubiquiti airOS config flow."""
+
+from typing import Any
+from unittest.mock import AsyncMock, patch
+
+from airos.exceptions import (
+    ConnectionAuthenticationError,
+    DataMissingError,
+    DeviceConnectionError,
+    KeyDataMissingError,
+)
+
+from homeassistant import config_entries
+from homeassistant.components.airos.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+
+async def test_form(
+    hass: HomeAssistant, mock_setup_entry: AsyncMock, ap_fixture: dict[str, Any]
+) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] is FlowResultType.FORM
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.airos.config_flow.AirOS",
+        autospec=True,
+    ) as airos_device_mock:
+        airos_device_mock.return_value.login.return_value = True
+        airos_device_mock.return_value.status.return_value = ap_fixture
```

Comment:
you can move this patch statement to `mock_airos_client` and then you don't need to have these patches at all